src/daemon/config.rs

```diff
@@ -578,6 +587,8 @@ impl Config {
             None
         };
 
+        let suspend_child_after_inactive_seconds = if enable_suspend { Some(3) } else { None };
```

Comment:
Should this be a constant or configurable (instead of false use None/Some)?

Reply:
it could be prettier, but it's not really in the way right now and requires refactoring many tests. So I prefer to just leave it. I used a bool because it felt consistent with 'enable_testbed' and 'enable_ca_refresh'.